.github/CODEOWNERS

```
@@ -0,0 +1,2 @@
+# Maintained by the MongoDB APIx-Integrations team
+* @mongodb-terraform-modules/APIx-Integrations
```

```suggestion
* @terraform-mongodbatlas-modules/APIx-Integrations
```
the GH UI was showing the team was not being found